### DISCUSSION TITLE
Two-column views of commits to files

### DISCUSSION THREAD

**Discussion by grahamperrin** [community/community#23614] [New to GitHub]
<p>Until today, I imagined that this was a feature of Refined GitHub.</p>
<p>For example, from <a href="https://forums.freebsd.org/posts/548712" rel="noopener nofollow ugc">a post to FreeBSD Forums</a>:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/7/276ac945a9edf95f6076412839fc3ea39be988ca.png" data-download-href="/uploads/short-url/5CHpp8XO7Un8hFlHSlm7AQ40B86.png?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/192271/181078232-db61b19c-4b38-4052-bc08-998c3d849d2b.png" alt="" data-base62-sha1="5CHpp8XO7Un8hFlHSlm7AQ40B86" width="550" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/7/276ac945a9edf95f6076412839fc3ea39be988ca_2_550x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/7/276ac945a9edf95f6076412839fc3ea39be988ca_2_825x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/7/276ac945a9edf95f6076412839fc3ea39be988ca_2_1100x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/7/276ac945a9edf95f6076412839fc3ea39be988ca_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename"></span><span class="informations">1200×1090 173 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<ul>
<li>pre-commit to the left</li>
<li>committed to the right, in green.</li>
</ul>
<p>Today I realised, there’s just one column when I view <a href="https://github.com/freebsd/freebsd-ports/commit/352af02b389202ac426ecf026ac65bff7c61bd41#diff-01b82cb561ad6c59937469270ae848f3d63dc40228678b4a1ef02cdf48ecc889" rel="noopener nofollow ugc">the same GitHub URL</a> <strong>in a private window</strong> (without signing in):</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/d/1d08e417644e6e199aff289be078c1841dea2cf5.png" data-download-href="/uploads/short-url/48QTweTPE1GirJb8qzxoKOYCpQF.png?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/192271/181078233-3da4aaf9-3d3f-477f-a8e2-dfd8e46a0ec3.png" alt="" data-base62-sha1="48QTweTPE1GirJb8qzxoKOYCpQF" width="690" height="324" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/d/1d08e417644e6e199aff289be078c1841dea2cf5_2_690x324.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/d/1d08e417644e6e199aff289be078c1841dea2cf5_2_1035x486.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/d/1d08e417644e6e199aff289be078c1841dea2cf5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/d/1d08e417644e6e199aff289be078c1841dea2cf5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename"></span><span class="informations">1252×589 86.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Is the two column view a feature of GitHub?</p>
<p>Not presented to the public (without signing in)?</p>

**Comment by jfullstackdev** [comment]
<p>nope, I click the link using incognito, not signed in,</p>
<p>you both have the view, just either click, <code>Split</code> or <code>Unified</code></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/8/382b3131ed1bd9bcee310263d5f296f515c0c9c9.png" data-download-href="/uploads/short-url/80ThbjQGzVbwO2JTNCPstDVyzyx.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078234-3d68eef8-7ebf-4462-9353-5c66f7bf00de.png" alt="image" data-base62-sha1="80ThbjQGzVbwO2JTNCPstDVyzyx" width="690" height="311" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/8/382b3131ed1bd9bcee310263d5f296f515c0c9c9_2_690x311.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/8/382b3131ed1bd9bcee310263d5f296f515c0c9c9_2_1035x466.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/8/382b3131ed1bd9bcee310263d5f296f515c0c9c9_2_1380x622.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/8/382b3131ed1bd9bcee310263d5f296f515c0c9c9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1660×750 48.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by grahamperrin** [comment, marked as answer]
<p>Thanks!</p>
<p>So I can manually edit the URL, for other readers to get what’s required <strong>immediately</strong>.</p>
<p><code>?diff=split</code> in the midst of this edition of the URL:</p>
<p><a href="https://github.com/freebsd/freebsd-ports/commit/352af02b389202ac426ecf026ac65bff7c61bd41?diff=split#diff-01b82cb561ad6c59937469270ae848f3d63dc40228678b4a1ef02cdf48ecc889" rel="noopener nofollow ugc">https://github.com/freebsd/freebsd-ports/commit/352af02b389202ac426ecf026ac65bff7c61bd41?diff=split#diff-01b82cb561ad6c59937469270ae848f3d63dc40228678b4a1ef02cdf48ecc889</a></p>
<p>Without this, if I’m not mistaken:</p>
<ol>
<li>visually remember what’s on screen</li>
<li>key <kbd>Home</kbd>, or scroll up until the Split button becomes visible</li>
<li>click the button</li>
<li>as a different page loads, lose the anchor</li>
<li>scroll down to the altered view of what was memorised from the previous page.</li>
</ol>
<p>I fixed the link in FreeBSD Forums.</p>
